lib/eventboss/queue_listener.rb

```diff
@@ -3,9 +3,32 @@ class QueueListener
     class << self
       def list
```

Comment:
should list be somehow made private? only list active will be used outside i guess?